src/store/mutationNames.js

```diff
@@ -0,0 +1 @@
+// export const NAME = ''
```

Comment:
Hm now I'm not sure if it's good idea i.e. I like that you want to use seperate constants to keep mutations names but a seperate file IMO is overhead. IMO if we get to much content in mutations file we can always separate the store to different modules. And we probably will do it later cause project seems kinda big. What do you think?

Reply:
ok